linux-setup.sh

```diff
@@ -53,6 +53,31 @@ install_go() {
     fi
 }
 
+# Builds and installs `mkcert` which is used by the following things in
+# webapp:
+# - https://khanacademy.dev
+# - the "Vitejs Directly" option in the dev support bar
+#
+# NOTE: This depends on `go` being installed.
+install_mkcert() {
+    if ! which mkcert >/dev/null; then
+        update "Installing mkcert..."
+        builddir=$(mktemp -d -t mkcert)
```

Comment:
Sorry kevin this doesn't work -- you need at least three X's:
```
builddir=$(mktemp -d -t mkcert.XXXXX)
```

Reply:
I'll make that change as an audit.  The script seemed to work on MacOS so there might be some differences in `mktemp` behavior between the systems.  Thanks for catching this.